examples/stable_diffusion_v2/ldm/modules/train/callback.py

```diff
@@ -141,7 +135,7 @@ def on_train_step_end(self, run_context):
                     append_dict={
                         "epoch_num": cur_epoch,
                         "cur_step": cur_step,
-                        "loss_scale": loss_scale_manager.get_loss_scale(),
+                        "loss_scale": cb_params.train_network.scale_sense.asnumpy().item(),
```

Comment:
how is the result different using this api?

Reply:
so far `loss_scale_manager.get_loss_scale()` will only return the initial assignment of the loss scale, not return the real value during training process. I think it is a bug.